CHANGELOG.md

```diff
@@ -1,6 +1,28 @@
 NativeScript CLI Changelog
 ================
 
+1.2.2 (2015, August 7)
+==
+
+### New
+* [Implemented #470](https://github.com/NativeScript/nativescript-cli/issues/470): Add timeout option to debug command.
+* [Implemented #682](https://github.com/NativeScript/nativescript-cli/issues/682): Set `--debug-brk` as default option of `tns debug` command.
+* [Implemented #706](https://github.com/NativeScript/nativescript-cli/issues/706): LiveSync to iOS Simulator devices running iOS 8.x.
+
+### Fixed
+
+* [Fixed #470](https://github.com/NativeScript/nativescript-cli/issues/470): Command failed due to space in file path.
+* [Fixed #632](https://github.com/NativeScript/nativescript-cli/issues/632): Android debugger does not work on some OSX systems.
+* [Fixed #652](https://github.com/NativeScript/nativescript-cli/issues/652): Debug command for iOS does not load inspector from the added ios-runtime.
+* [Fixed #663](https://github.com/NativeScript/nativescript-cli/issues/663): Verify that Java is at least 1.7.
+* [Fixed #671](https://github.com/NativeScript/nativescript-cli/issues/671): iOS debugger doesn't work for projects created with framework version lower than 1.2.0.
+* [Fixed #679](https://github.com/NativeScript/nativescript-cli/issues/679): Library add command adds the framework with its full path into xcode project.
```

Comment:
Ok, let's make this 

`tns library add ios` adds the framework with its full path in the Xcode project.